ios/engine/KMEI/KeymanEngine/Classes/Resource Management/AssociatingPackageInstaller.swift

```diff
@@ -156,7 +155,7 @@ public class AssociatingPackageInstaller<Resource: LanguageResource, Package: Ty
         if complete {
           progressCallback(.complete)
         } else {
-          progressCallback(.inProgress)
+          //progressCallback(.inProgress)
```

Comment:
Is this commented out for future feature (comment at l. 191)?

Reply:
Yes.